proto/stakeibc/gov.proto

```diff
@@ -15,3 +15,19 @@ message AddValidatorProposal {
   string validator_name = 4; 
   string validator_address = 5 [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
 }
+
+
+message AddValidatorProposalWithDeposit {
```

Comment:
Can we remove the old struct above and rename this to `AddValidatorProposal`?

Reply:
We need both (the one without deposit is used in line 81). This is following convention from community-pool-spend proposals